geode-core/src/main/java/org/apache/geode/internal/cache/backup/BackupDestination.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.internal.cache.backup;
+
+import java.io.IOException;
+
+public interface BackupDestination {
+  String USER_FILES_DIRECTORY = "user";
```

Comment:
Why does the BackupDestination interface have all these String constants? They seem like an implementation details that someone implementing this interface would not care about.

Reply:
If we get the BackupDefinition to contain only a list of files, then these would not be needed here (and could be moved into that class). These would be used to set the correct relative paths. We should consider that.